static/app/views/dashboardsV2/manage/dashboardCard.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ActivityAvatar from 'app/components/activity/item/avatar';
+import Card from 'app/components/card';
+import Link from 'app/components/links/link';
+import TextOverflow from 'app/components/textOverflow';
+import {t} from 'app/locale';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {User} from 'app/types';
+
+type Props = {
+  title: string;
+  detail: React.ReactNode;
+  to: React.ComponentProps<typeof Link>['to'];
+  renderWidgets: () => React.ReactNode;
+  createdBy?: User | undefined;
+  dateStatus?: React.ReactNode;
+  onEventClick?: () => void;
+};
+
+export function DashboardCard(props: Props) {
```

Comment:
should we remove this export as we are already using `export default DashboardCard`?

Reply:
oops, yeah debug code that I left in here